CHANGES.rst

```diff
@@ -107,6 +107,8 @@ API changes
 
 - ``astropy.wcs``
 
+  - Improved documentation of WCS.calc_footprint [#3551]
```

Comment:
Oops, sorry. I'll change it.
On Mar 4, 2015 1:28 PM, "Thomas Robitaille" <notifications@github.com>
wrote:

> In CHANGES.rst
> <https://github.com/astropy/astropy/pull/3557#discussion_r25757388>:
>
> > @@ -107,6 +107,8 @@ API changes
> >
> >  - ``astropy.wcs``
> >
> > +  - Improved documentation of WCS.calc_footprint [#3551]
>
> @vortex-ape <https://github.com/vortex-ape> - this is what I meant should
> be included in the changelog - the fact the ValueError is now changed
> (hence why it belongs in the API change section) - you indeed don't need to
> mention the doc change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/astropy/astropy/pull/3557/files#r25757388>.
>